pkg/roachprod/vm/gce/utils.go

```diff
@@ -31,7 +31,11 @@ import (
 const gceDiskStartupScriptTemplate = `#!/usr/bin/env bash
 # Script for setting up a GCE machine for roachprod use.
 
-set -x
+# ensure any failure fails the entire script
+set -eux
+
+# Redirect output to stdout/err and a log file
+exec &> >(tee {{ .StartupLogs }})
```

Comment:
What if a VM is restarted? It would be nice to append vs. overwrite. It would also be nice to have a timestamp as a frame of reference.